flake.nix

```
@@ -22,7 +22,13 @@
 
       checks = config.packages;
 
-      packages = import ./lib/packages.nix { inherit pkgs pkgs-unstable; custom-lib = self.lib; };
+      packages =
+        let
+          versions = builtins.fromJSON (builtins.readFile ./versions.json);
+          releases = import ./lib/packages.nix { inherit pkgs pkgs-unstable; custom-lib = self.lib; releases = versions.releases; };
+          latestVersions = builtins.mapAttrs (cycle: version: releases.${version}) versions.latest;
```

@oscar-izval the following proposal is a common pattern in Haskell, but I have also seen it in Nix: prefixing a variable with `_` to denote that it is not being referenced:

```suggestion
          latestVersions = builtins.mapAttrs (_: version: releases.${version}) versions.latest;
```